pkg/fanal/analyzer/language/analyze.go

```diff
@@ -105,11 +106,15 @@ func toApplication(fileType, filePath, libFilePath string, r dio.ReadSeekerAt, l
 			}
 			locs = append(locs, l)
 		}
+		libPath := libFilePath
+		if lib.FilePath != "" {
+			libPath = filepath.ToSlash(lib.FilePath)
```

Comment:
We use `/` in `PkgPath` when scanning in Windows.
That's why I added `ToSlash` here.

Reply:
Oh, we should have used slashes here. `path.Join` must be used rather than `filepath.Join`. The inner separator in ZIP files is always slashed even though it is created on Windows.

https://github.com/DmitriyLewen/go-dep-parser/blob/4888ccee8867aa2d59ad2966faf06944e808e8e3/pkg/java/jar/parse.go#L196

>   4.4.17 file name: (Variable)
>
>       4.4.17.1 The name of the file, with optional relative path.
>       The path stored MUST NOT contain a drive or
>       device letter, or a leading slash.  All slashes
>       MUST be forward slashes '/' as opposed to
>       backwards slashes '\' for compatibility with Amiga
>       and UNIX file systems etc.  If input came from standard
>       input, there is no file name field.  

https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT

Reply:
fixed in https://github.com/aquasecurity/go-dep-parser/pull/212